doc/deployment/deploy_w_tls.md

```diff
@@ -0,0 +1,36 @@
+# Deploy Pachyderm with TLS
+
+You can deploy your Pachyderm cluster with Transport Layer Security(TLS)
+enabled to ensure your cluster communications are protected from external
+attackers, and all the communication parties are verified by means of a
```

Comment:
```suggestion
attackers and all the communication parties are verified by means of a
```

Reply:
we need a comma there. "You can ..., all the communication parties" - two independent clauses in a compound sentence.